core/src/main/scala/org/http4s/syntax/AllSyntax.scala

```diff
@@ -4,6 +4,7 @@ package syntax
 trait AllSyntax
     extends AnyRef
     with AsyncSyntax
+    with ByteChunkSyntax
```

Comment:
One reason this was separate was because we don't own either the type class or the instance type.  Every time we've done this, we've said, "this missing instance/operation should be contributed upstream."  I think this should be an internal syntax until we can get rid of it, either by contributing it to fs2-scodec or dropping fs2-scodec.  (I still suspect we can do the latter.)

Reply:
Fair enough. Let's scrap the entire PR for now then.